assets/src/edit-story/components/library/panes/text/textSets/raw/cover.json

```diff
@@ -0,0 +1,3005 @@
+{
+  "current": "7cf5cb87-0090-4ef4-b047-a745f8f6bfbe",
+  "selection": [],
+  "story": {
+    "stylePresets": {
+      "colors": [
```

Comment:
It's just straight-up silly that we have all this irrelevant meta-data here, that's not at all applicable to the text set. We should find some way to clean these up by the end, but we can skip that for now until we have all the sets in and approved.

Reply:
Yeah, the concept was that one could simply copy the contents of the debug screen so Sam or another, non-dev, could make the presets how they wanted and simply copy the debug screen contents (without having to think about it) and it would work.  But we could definitely trim out the unneeded stuff.